packages/react-component-library/src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -86,3 +86,9 @@ RangeWithInitialValue.args = {
   initialStartDate: parseISO('2021-12-05'),
   initialEndDate: parseISO('2021-12-15'),
 }
+
+export const JumpToToday = Template.bind({})
```

Comment:
This feature previously existed but there was no dedicated story or prop to toggle it.